.github/workflows/gradle.yml

```diff
@@ -1,17 +1,98 @@
 name: Gradle build
 
-on: [push]
+on: [ push ]
 
 jobs:
-  build:
+  build-ubuntu:
+    runs-on: ubuntu-20.04
 
-    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install Chrome
+        run: |
+          sudo apt install -y libappindicator1 fonts-liberation
+          wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          sudo dpkg -i google-chrome*.deb
+      - name: Cache gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            **/.gradle
+            **/build
+          key: gradle_6_6_1
```

Comment:
It will break on gradle wrapper change. Is it possible to move gradle version to top level parameters?